.github/workflows/test.yaml

```diff
@@ -48,12 +48,29 @@ jobs:
       - name: Test
         run: make test
 
+  nix-build:
+    needs: static
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+      fail-fast: false
+    
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: cachix/install-nix-action@v30
+
+      - run: nix build --verbose -L .#bootstrap-srv
```

Comment:
Quite a bit slower than other CI checks, can we add cachix?

Reply:
yes